src/radio-statuses/observable.js

```diff
@@ -0,0 +1,23 @@
+// used in jsdoc
+// eslint-disable-next-line no-unused-vars
+import { Observable } from 'rxjs';
+import { fetchRadioStatuses } from './fetch';
+import { refreshRateInSeconds } from './refresh-rate';
+
+/**
+ * @type {Observable<RadioStatus[]>}
+ */
+
+const interval = refreshRateInSeconds * 1000;
```

Comment:
```suggestion
const interval = refreshRateInSeconds * 1000;

/**
 * @type {Observable<RadioStatus[]>}
 */
```

Jsdoc musi być bezpośrednio nad zmienną